### PR TITLE
Rn UI

### PR DESCRIPTION
**Pull request by subhamtsd**
- [x]  Fixed Layout sizing bug
- [x] Routing till 2nd Label is achieved
- [x] Worked on some UI Demonstration specially for RN compossible UI 
- [x] Removed the API integration part from the RN compossible repo

**First Component**
<hr>

First Component

**RnComposibleUI Dashboard Component**
<hr>

RnComposibleUI Dashboard Component

### CODE REVIEW

**Comment by saurshaz** [comment]
@subhamtsd  pls fix the lint errors

**Comment by saurshaz** [comment]
@subhamtsd Pls lint the code

File: src/pages/components/json-form/ConnectedForm.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```
Comment (saurshaz) [inline src/pages/components/json-form/ConnectedForm.tsx:1]: pls deletge this one

File: src/state-mgmt/reducers.ts [hunk 2]
```diff
@@ -64,10 +66,31 @@ const user = (state = { ...initialState.user }, action) => {
 const env = () => {
   return environment;
 };
+
+/*
```
Comment (saurshaz) [inline src/state-mgmt/reducers.ts:70]: This shall not be there

File: src/uiConfig/index.tsx [hunk 3]
```diff
@@ -7,10 +8,20 @@ import { AddEditEntity } from "../pages/components/AddEditEntity";
 import { ListEntities } from "../pages/components/ListEntities";
 import { ShowEntity } from "../pages/components/ShowEntity";
 import SearchList from "../pages/components/SearchList";
-// import {MainApp} from '../pages/screens/MainApp'
+import { OrderLineView } from "../pages/components/OrderLineView";
+
+// TSD WEB APP INTEGRATION
+import { HeaderComponent } from "../pages/components/HeaderComponent";
+import { OrderSearchList } from "../pages/components/OrderSearchList";
```
Comment (saurshaz) [inline src/uiConfig/index.tsx:15]: Order Search related parts , remove them

File: src/utils/event.config.ts [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+export const orderLineViewUpdateEventConfig: any = {
+  "OrderLineView.buttonRef": {
```
Comment (saurshaz) [inline src/utils/event.config.ts:3]: This can be deleted from the commit

File: src/pages/components/RenderList.tsx [hunk 5]
```diff
@@ -1,10 +1,11 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
 import React, { useState } from "react";
 import {
```
Comment (saurshaz) [inline src/pages/components/RenderList.tsx:3]: 01⁄05⁄212021-01-05Tuesday 5_1

File: src/pages/components/OrderSearchList/index.tsx [hunk 6]
```diff
@@ -0,0 +1,113 @@
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from "react";
+import {
+  ActivityIndicator,
+  Dimensions,
+  ScrollView,
+  StyleSheet,
```
Comment (saurshaz) [inline src/pages/components/OrderSearchList/index.tsx:7]: 01⁄05⁄212021-01-05Tuesday 5
delete these

**Review by saurshaz — CHANGES_REQUESTED** [review]
- requested minor cleanup (look at review comments)

- in addition to the comments, remove all console.logs (unless there is a need, add a TODO, where it's needed to stay)
- also remove all uncommented code (unless there is a need, add a TODO, where it's needed to stay)

File: src/state-mgmt/reducers.ts (hunk 2 above)
Comment (subhamtsd) [reply src/state-mgmt/reducers.ts:70]: Hi @saurshaz 

Should I remove the env part ?

File: src/uiConfig/index.tsx (hunk 3 above)
Comment (subhamtsd) [reply src/uiConfig/index.tsx:15]: Removed the tsd part component

File: src/utils/event.config.ts (hunk 4 above)
Comment (subhamtsd) [reply src/utils/event.config.ts:3]: Deleted

File: src/pages/components/RenderList.tsx (hunk 5 above)
Comment (subhamtsd) [reply src/pages/components/RenderList.tsx:3]: Deleted the Render List

File: src/pages/components/OrderSearchList/index.tsx (hunk 6 above)
Comment (subhamtsd) [reply src/pages/components/OrderSearchList/index.tsx:7]: Deleted

File: src/state-mgmt/reducers.ts (hunk 2 above)
Comment (saurshaz) [reply src/state-mgmt/reducers.ts:70]: you can , if it is not used

**Review by saurshaz — APPROVED** [review]
Good to merge , @subhamtsd to push the lint fixes soon

**Review by saurshaz — APPROVED** [review]

**Pull request merged**